content/docs/start/metrics-parameters-plots.md

```diff
@@ -133,14 +136,15 @@ $ git add scores.json prc.json roc.json
 $ git commit -a -m "Create evaluation stage"
 ```
 
-Later we will see how these and other can be used to compare and visualize
-different pipeline iterations. For now, let's see how can we capture another
-important piece of information that will be useful for comparison: parameters.
+Later we will see how to
+[compare and visualize different pipeline iterations](#comparing-iterations).
+For now, let's see how can we capture another important piece of information
+which will be useful for comparison: parameters.
 
 ## Defining stage parameters
 
 It's pretty common for data science pipelines to include configuration files
-that define adjustable parameters to train a model, do pre-processing, etc. DVC
+which define adjustable parameters to train a model, do pre-processing, etc. DVC
```

Comment:
"That" was better here. With "which" it seems like we're suddenly talking about configuration files in general, and it's not true that all config files adjust parameters. Or we can go with "to":
```suggestion
to define adjustable parameters to train a model, do pre-processing, etc. DVC
```